client/src/ui/components/Alert.tsx

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import cx from 'classnames';
+
+interface AlertProps {
+  children?: React.ReactNode;
+  className?: string;
+  status: 'info' | 'warning' | 'error';
+}
+
+export const Alert = ({
+  children,
+  className,
+  status,
+  ...props
+}: AlertProps) => (
+  <div
+    {...props}
+    className={cx(
+      'px-4 py-3 shadow-md rounded-b border-t-4',
+      { 'bg-blue-100 text-blue-900 border-blue-500': status === 'info' },
+      {
+        'bg-orange-100 text-orange-700 border-orange-500': status === 'warning',
```

Comment:
@pradel maybe we can do yellow for this one? My initial impression was that warning and error is quite similar to each other. What do you think?